src/device-registry/routes/v1/activities.js

```diff
@@ -0,0 +1,217 @@
+const express = require("express");
+const router = express.Router();
+const constants = require("@config/constants");
+const mongoose = require("mongoose");
+const ObjectId = mongoose.Types.ObjectId;
+const activityController = require("@controllers/create-activity");
```

Comment:
each route files for each use-case now has its own file. Neat stuff I guess? :) @BenjaminSsempala